### ISSUE TITLE
list() ist not backwards compatible

### ISSUE THREAD

**Issue by theova** [AuditeMarlow/base16-manager#26]
In commit e73ae11269a4232b25d5c50e0c91d2b22344573e the function `list()` changed to read from the file `$CONFIG_PATH/installed`:
https://github.com/AuditeMarlow/base16-manager/blob/caa01cb917595c6f62a4adcba66f0aeaca1f4375/base16-manager#L85

If templates were installed before this commit, they aren't included in this file or the file doesn't even exist.

A possible workaround could be
```
if ! file_exits "$CONFIG_PATH/installed"; then
	ls "$CONFIG_PATH" > "$CONFIG_PATH/installed"
fi
```

but then I don't see the difference to directly return the output of
```
ls "$CONFIG_PATH" > "$CONFIG_PATH/installed"
```

So, what's the gain of storing installed templates in a seperate file?
How can we make this function backwards compatible again?

**Comment by AuditeMarlow** [comment]
It's especially complicated because the installed templates aren't simply displayed by doing an `ls` on the base16-manager folder, as the templates go down 1 level from there and I'd like to preserve the maintainers too.

**Comment by theova** [comment]
See PR #28.

Probably we can simplify the other functions using `$CONFIG_PATH/installed` as well?

**Comment by AuditeMarlow** [comment]
Resolved by #28.
